vagrant/provisioning/roles/snowbound/tasks/main.yml

```diff
@@ -47,7 +47,7 @@
           catalina_out: "{{ root_folder }}/log/snowbound/catalina.out"
           catalina_temp: "{{ root_folder }}/tmp/snowbound"
           log_folder: "{{ root_folder }}/log/snowbound"          
-          java_home: "/usr/lib/jvm/java-1.8.0-openjdk"
+          java_home: "{{ '/usr/lib/jvm/java-1.8.0-openjdk' if snowbound_vendor_version == '4.1' else '/usr/lib/jvm/java-11-openjdk' }}"
```

Comment:
Is '4.1' == '4.10'?  Otherwise looks great

Reply:
... or consider version comparison using version() function, https://docs.ansible.com/ansible/latest/user_guide/playbooks_tests.html#version-comparison

Reply:
It is 4.10 thanks! Updated that in the latest commit.
version() function can be used here. I see we are using it to compare greater than, less than...